compiler/ballerina-lang/src/main/java/io/ballerina/projects/JarResolver.java

```diff
@@ -86,27 +98,35 @@ public List<PlatformLibrary> providedPlatformLibs() {
         return providedPlatformLibs;
     }
 
-    // TODO These method names are too long. Refactor them soon
     public Collection<JarLibrary> getJarFilePathsRequiredForExecution() {
+        return getJarFilePathsRequiredForExecution(false);
+    }
+
+    // TODO These method names are too long. Refactor them soon
```

Comment:
I don't see an issue with this method name. Having long name which explain the purpose is better than having a short name which hardly explain what the method does.

Reply:
That comment was already there in the legacy code. Seems like git got confused because I did changes near that line. Anyways I will keep in mind the method naming conventions you mentioned.